docs/about/index.rst

```diff
@@ -8,6 +8,29 @@ About
 Mission
 -------
 
+Fairlearn is an open source toolkit that seeks to empower data scientists and
+developers to assess and improve fairness of their AI systems.
+The toolkit aspires to include:
+
+- Fairness metrics and unfairness mitigation algorithms
+- Interactive dashboard for fairness assessment
+- Educational materials (comprehensive user guide, detailed case studies,
+  white papers, etc.)
+
+Development of Fairlearn is firmly grounded in the understanding that fairness
+in AI systems is a sociotechnical challenge.  Because there are many complex
+sources of unfairness&mdash;some societal and some technical&mdash;it is not
+possible to fully "debias" a system or to guarantee fairness.
```

Comment:
It was perfectly clear to me, but I'm happy to take suggestions.

Reply:
Let's hold off on micro-edits until we get the content down. This specific sentence was taken from the whitepaper and went through several editorial rounds (that doesn't mean it cannot be improved further of course...). When it comes to text--my suggestion is to focus on the clarity issues for now.

If there are remaining stylistic issues that you feel strongly about, it might be better to resolve those on a call -- @hannawallach had specific reasons for writing this sentence as it is written, so we should ensure that the intent is not lost in the rewrite.